pkg/registry/common/localbypass/find_server.go

```diff
@@ -32,5 +35,9 @@ func (s *localBypassNSEFindServer) Send(endpoint *registry.NetworkServiceEndpoin
 	if u, ok := s.nseURLs.Load(endpoint.Name); ok {
 		endpoint.Url = u.String()
 	}
+	if endpoint.Url == s.nsmgrURL {
```

Comment:
Such approach is invalid because it breaks streaming find in `discover`. It can be tested with:
1. Add some delay for the registry interactions in `pkg/registry/chains/memory/server.go`:
   ```go
   ...
   nseChain := chain.NewNetworkServiceEndpointRegistryServer(
   	checknse.NewServer(nil, func(*testing.T, *registry.NetworkServiceEndpoint) {
   		time.Sleep(100 * time.Millisecond)
   	}),
   	expire.NewNetworkServiceEndpointRegistryServer(ctx, expiryDuration),
   	...
   )
   ...
   ```
2. Postpone endpoint startup for `TestNSMGR_LocalUsecase` test:
   ```go
   counter := &counterServer{}
   time.AfterFunc(time.Second, func() {
   	_, err := domain.Nodes[0].NewEndpoint(ctx, nseReg, sandbox.GenerateTestToken, counter)
   	require.NoError(t, err)
   })
   ```
Tests sometimes fails with:
```
=== RUN   TestNSMGR_LocalUsecase
No logger in context
No logger in context
No logger in context
    server_test.go:394: 
        	Error Trace:	server_test.go:394
        	Error:      	Received unexpected error:
        	            	rpc error: code = Unknown desc = EOF
        	            	Error returned from api/pkg/api/networkservice/networkServiceClient.Request
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*beginTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:62
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:69
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:71
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*endTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:99
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:71
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/common/authorize.(*authorizeClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/common/authorize/client.go:53
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*beginTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:58
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:69
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:69
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*endTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:99
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:71
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/utils/metadata.(*metaDataClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/utils/metadata/client.go:44
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*beginTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:58
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:69
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:69
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*endTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:99
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:71
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/common/refresh.(*refreshClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/common/refresh/client.go:58
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*beginTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:58
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:69
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:69
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*endTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:99
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:71
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/common/heal.(*healClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/common/heal/client.go:50
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*beginTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:58
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:69
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:69
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/trace.(*endTraceClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/trace/client.go:99
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextClient).Request
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/core/next/client.go:71
        	            	github.com/networkservicemesh/sdk/pkg/networkservice/common/serialize.(*serializeClient).Request.func1
        	            		/Users/user/Projects/NSM/sdk/pkg/networkservice/common/serialize/client.go:44
        	            	github.com/networkservicemesh/sdk/pkg/tools/multiexecutor.(*MultiExecutor).AsyncExec.func2.1
        	            		/Users/user/Projects/NSM/sdk/pkg/tools/multiexecutor/multi_executor.go:54
        	            	github.com/edwarnicke/serialize.(*Executor).process
        	            		/Users/user/go/pkg/mod/github.com/edwarnicke/serialize@v1.0.7/serialize.go:68
        	Test:       	TestNSMGR_LocalUsecase
No logger in context
--- FAIL: TestNSMGR_LocalUsecase (1.12s)
```